launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml

```diff
@@ -152,11 +155,16 @@
   <!-- CenterPoint -->
   <group if="$(eval &quot;'$(var lidar_detection_model)'=='centerpoint'&quot;)">
     <push-ros-namespace namespace="centerpoint"/>
-    <include file="$(find-pkg-share lidar_centerpoint)/launch/lidar_centerpoint.launch.xml">
-      <arg name="input/pointcloud" value="$(var input/pointcloud)"/>
-      <arg name="output/objects" value="objects"/>
-      <arg name="score_threshold" value="$(var score_threshold)"/>
-    </include>
+    <group>
```

Comment:
[Q] Why do you need group tag again? (Group tag is already set in L156)

Reply:
Group tags are used in `lidar_based_detection.launch.xml` but not in `camera_lidar_based_detection.launch.xml` for centerpoint.
So I unified to use for https://github.com/autowarefoundation/autoware.universe/pull/1876

Reply:
I don't think either of these are necessary... but it's ok